### PR TITLE
Also pass XF86 Power Off and Power Down keys through when using --pass-media-keys

### PR DESCRIPTION
**Pull request by Raymo111**
I find it very annoying that I have to unlock before I can sleep my computer. This should not have any additional security risks than the other media keys.

### CODE REVIEW

**Comment by PandorasFox** [comment]
While I'm for passing these two keys, I'd rather do this one of two ways:

* rename the arg to something like "pass xf86 keys"
* have a separate arg for the power keys

The main reasons being that the power key(s) aren't really media keys, and it changes the meaning of the existing arg.

(I think the former option isn't as great, since it would also inherently mean removing the current arg & also impacts existing workflows).

Otherwise looks good.

**Comment by Raymo111** [comment]
How about we make 3 separate options?
1. Pass xf86 media keys
2. Pass xf86 brightness keys
3. Pass xf86 power keys
So then if we ever want to pass more keys we can always go back and add another option? Just because technically brightness isn't media either...

**Comment by PandorasFox** [comment]
Sounds good. I'd forgotten that the brightness keys were passed too.

**Comment by Raymo111** [comment]
Ok, I've made it work.

File: i3lock.c [hunk 1]
```diff
@@ -1391,8 +1412,12 @@ int main(int argc, char *argv[]) {
         {"radius", required_argument, NULL, 402},
         {"ring-width", required_argument, NULL, 403},
 
-        // alignment
+		// pass keys
+        {"pass-media-keys", no_argument, NULL, 'm'},
+        {"pass-screen-keys", no_argument, NULL, 's'},
```
Comment (PandorasFox) [inline i3lock.c]: s and p are both already used ("line-uses-inside" and "pointer" above) - it's fine to just leave them with a numeric value in the 900s to switch on.

Comment (Raymo111) [reply i3lock.c]: Those are using capital letters, no? I remember doing a search for those flags and they didn't come up.

Comment (PandorasFox) [reply i3lock.c]: unfortunately not - check lines 1405 and 1375 above. ('s' and 'S' are both already in use)

There's generally too many args to reliably give meaningful short arg names to any new args - plus, most people just run i3lock through a script that holds all the args, so short args aren't really necessary anyhow.

Comment (Raymo111) [reply i3lock.c]: True. Have removed them all. 600s were available so I used those.

Comment (PandorasFox) [reply i3lock.c]: Perfect. Thank you very much!

Comment (Raymo111) [reply i3lock.c]: Cool, no worries!

**Pull request merged**